idaes/gas_solid_contactors/unit_models/tests/test_BFB.py

```diff
@@ -243,28 +244,31 @@ def test_solve(self, iron_oc):
     @pytest.mark.skipif(solver is None, reason="Solver not available")
     @pytest.mark.component
     def test_solution(self, iron_oc):
-        assert (pytest.approx(0.14, abs=1e-2) ==
+        assert (pytest.approx(0.168, abs=1e-2) ==
                 iron_oc.fs.unit.velocity_superficial_gas[0, 0].value)
-        assert (pytest.approx(1.052, abs=1e-2) ==
+        assert (pytest.approx(1.601, abs=1e-2) ==
                 iron_oc.fs.unit.velocity_superficial_gas[0, 1].value)
         assert (pytest.approx(0.015, abs=1e-2) ==
                 iron_oc.fs.unit.bubble_diameter[0, 0].value)
-        assert (pytest.approx(1.04, abs=1e-2) ==
+        assert (pytest.approx(1.266, abs=1e-2) ==
                 iron_oc.fs.unit.bubble_diameter[0, 1].value)
-        assert (pytest.approx(0.375, abs=1e-2) ==
+        assert (pytest.approx(0.417, abs=1e-2) ==
                 iron_oc.fs.unit.velocity_bubble[0, 0].value)
-        assert (pytest.approx(3.290, abs=1e-2) ==
+        assert (pytest.approx(4.067, abs=1e-2) ==
                 iron_oc.fs.unit.velocity_bubble[0, 1].value)
-        assert (pytest.approx(0.267, abs=1e-2) ==
-                iron_oc.fs.unit.delta[0, 0].value)
         assert (pytest.approx(0.307, abs=1e-2) ==
+                iron_oc.fs.unit.delta[0, 0].value)
+        assert (pytest.approx(0.384, abs=1e-2) ==
                 iron_oc.fs.unit.delta[0, 1].value)
-        assert (pytest.approx(124169.5118, abs=1e-2) ==
+        assert (pytest.approx(100671.070, abs=1e-2) ==
+                iron_oc.fs.unit.gas_outlet.pressure[0].value)
+        assert (pytest.approx(85328.9301, abs=1e-2) ==
+                iron_oc.fs.unit.gas_inlet.pressure[0].value -
                 iron_oc.fs.unit.gas_outlet.pressure[0].value)
```

Comment:
These tests results (and potentially some other tests results) will change when you update the orifice pressure drop to 3.4E3 Pa.